ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -241,6 +241,47 @@ tests:
       DISCONNECTED: "false"
       IP_STACK: v4
     workflow: baremetalds-assisted-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-multinode-spoke-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+    env:
+      ASSISTED_ZTP_CONFIG: |
+        SPOKE_CONTROLPLANE_AGENTS=3
+      DEVSCRIPTS_CONFIG: |
+        CLUSTER_NAME="ostest-master"
+        NUM_EXTRA_WORKERS=3
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=32768
+        EXTRA_WORKER_DISK=20
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
+      DISCONNECTED: "false"
+      IP_STACK: v4
+    workflow: baremetalds-assisted-operator-ztp
+- as: e2e-metal-assisted-operator-ztp-multinode-spoke
+  steps:
+    cluster_profile: packet
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:candidate
+    env:
+      ASSISTED_ZTP_CONFIG: |
+        SPOKE_CONTROLPLANE_AGENTS=3
+      DEVSCRIPTS_CONFIG: |
+        CLUSTER_NAME="ostest-master"
```

Comment:
Is the `CLUSTER_NAME` mandatory?

Reply:
This was a workaround for [BZ-1984155](https://bugzilla.redhat.com/show_bug.cgi?id=1984155). I will try without it because it seems like this kubeconfig directory is hardcoded somewhere.